### ISSUE TITLE
react/jsx-indent rule wrongly raises error in a special case

### ISSUE THREAD

**Issue by MaxInMoon** [eslint/eslint#9921]
**Tell us about your environment**

* **ESLint Version: 4.16.0**
* **Node Version: 8.6.0**
* **npm Version: 5.6.0**

**What parser (default, Babel-ESLint, etc.) are you using?**
"babel-eslint": "^7.1.1"

**Please show your full configuration:**

```js
    "babel-core": "^6.24.0",
    "babel-eslint": "^7.1.1",
    "eslint": "^4.16.0",
    "eslint-config-airbnb": "^14.1.0",
    "eslint-plugin-import": "^2.2.0",
    "eslint-plugin-jsx-a11y": "^4.0.0",
    "eslint-plugin-react": "^6.10.2",

     __

     "eslintConfig": {
       "parser": "babel-eslint",
       "extends": "airbnb",
     }
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```jsx


<div>
  {!isLoading &&
    <div />
  }
  {!isLoading &&
    <FloatingActionButton
      foo="foo"
    />
  }
</div>
```

```bash
  "eslint": "eslint components src test tools postcss.config.js webpack.config.js",
```

**What did you expect to happen?**

No lint error

**What actually happened? Please include the actual, raw output from ESLint.**

`< FloatingActionButton  />` component raises the lint error, but not the div above, at the same level.
This this the cli log:
` 177:13  error    Expected indentation of 10 space characters but found 12  react/jsx-indent`

**Comment by not-an-aardvark** [comment]
Thanks for the report. The `react/jsx-indent` rule is part of [`eslint-plugin-react`](https://github.com/yannickcr/eslint-plugin-react), which is not maintained by the ESLint team -- please report an issue on that repository instead.
